mmtrack/models/sot/__init__.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .prdimp import Prdimp
 from .siamrpn import SiamRPN
 from .stark import Stark
 
-__all__ = ['SiamRPN', 'Stark']
+__all__ = ['SiamRPN', 'Stark', 'Prdimp']
```

Comment:
Prdimp or PrDIMP?

Reply:
We rename all names containing `prdimp` as `Prdimp`